helm/prometheus-rules/templates/recording-rules/cortex.rules.yml

```diff
@@ -212,6 +212,13 @@ spec:
       record: aggregation:dex_requests_status_ok
     - expr: grafana_analytics_sessions_total
       record: aggregation:grafana_analytics_sessions_total
+      labels:
+        user_locale: "none"
```

Comment:
Should we not replace it with "" or null ?

Reply:
Doc says "A label with an empty label value is considered equivalent to a label that does not exist."
source: https://prometheus.io/docs/concepts/data_model/#metric-names-and-labels